backend/services/academics/my_courses.py

```diff
@@ -81,7 +85,7 @@ def _group_by_term(self, entities: list[SectionMemberEntity]) -> list[TermOvervi
                         subject_code=course.subject_code,
                         number=course.number,
                         title=memberships[0].section.override_title or course.title,
-                        role=memberships[0].member_role.name,
+                        role=memberships[0].member_role.value,
```

Comment:
Let's use the names here (if we need pretty casing, perhaps use a conversion function)